src/main/java/gyro/google/compute/NetworkResource.java

```diff
@@ -109,12 +111,22 @@ public void setId(String id) {
         this.id = id;
     }
 
+    @Output
+    public String getSelfLink() {
+        return selfLink;
+    }
+
+    public void setSelfLink(String selfLink) {
+        this.selfLink = selfLink;
+    }
+
     @Override
     public void copyFrom(Network network) {
         setId(network.getId().toString());
         setRoutingMode(network.getRoutingConfig().getRoutingMode());
         setDescription(network.getDescription());
         setName(network.getName());
+        setSelfLink(network.getSelfLink());
```

Comment:
Same as above.

Reply:
Sure I'll remove this as it's already covered in #51